server/static/js/device_status/device_status_page_highcharts.js

```diff
@@ -0,0 +1,252 @@
+'use strict';
+/**
+ * Basic style setting for highcharts
+ */
+Highcharts.setOptions({
+        chart: {
+            backgroundColor: '#363430',
+            borderRadius: 5
+        },
+        yAxis: {
+            labels: {
+                style: {
+                    color: '#EE5253'
+                }
+            },
+            title: {
+                style: {
+                    color: '#fff'
+                }
+            }
+        },
+        xAxis: {
+            labels: {
+                style: {
+                    color: '#EE5253'
+                }
+            },
+            title: {
+                style: {
+                    color: '#fff'
+                }
+            }
+        },
+        title: {
+            style: {
+                color: '#fff'
+            }
+        }
+    });
+
+$(document).ready(function () {
+    let deviceName = window.location.pathname.split("/").pop();
+    let activeTimespan = null;
```

Comment:
This variable actually looks redundant, since it only points to local URL variable. Not to mention that it doesn't really provide the active timespan - it's a URL container.
Would it make more sense to use that URL variable instead?